main.py

```diff
@@ -20,10 +20,7 @@
 
 class SettingsManager(object):
 	def __init__(self):
-		self.reset()
-
-	def reset(self):
-		self.settings = {
+		self.defaults = {
```

Comment:
This should probably be a function. so we can auto-generate path information based on if a file exists or not!

(say: auto-pick "./bios.bin" if that file exists, otherwise kept empty or `None` [the "/path/to/bios.bin" stuff should be exclusively handled by UI for unset fields imo])